### ISSUE TITLE
Describe app does not work

### ISSUE THREAD

**Issue by jeffmaury** [redhat-developer/vscode-openshift-tools#264]
I tried the `describe` command and got the following output:

```
Microsoft Windows [version 10.0.17134.345]
(c) 2018 Microsoft Corporation. Tous droits réservés.

@ C:\Users\JeffMAURY\AppData\Local\Programs\Microsoft VS Code
> odo project set myproject; odo app describe springboot
Error: accepts 1 arg(s), received 5
Usage:
  odo project set [flags]

Examples:
  # Set the current active project
  odo project set myproject
```

**Comment by jrichter1** [comment]
It's the semicolon. On Windows it is used to separate arguments instead of commands. And the same problem is in a few other odo commands.

**Comment by dgolovin** [comment]
Resolved with #265.

**Comment by jeffmaury** [comment]
Watch command still use `;`

**Comment by dgolovin** [comment]
Resolved via dd15833.
